namespaceResourceQuotaResizing/nsQuotaResizing4_executableContainerResizing/README.md

```diff
@@ -0,0 +1,34 @@
+# Executable Container Resizing
+
+This scenario is to test the executable part of container resizing actions due to namespace resource quota constraint.
+
+## Container Executable Resizing Up to the Best Desired Value
+
+First create such namespace with very large resource quota (or without resource quota):
+
+```
+Namespace:
+- CPU limit quota capacity: 5000m
+
+- ContainerSpec:
+    - Container:
+        - CPU limit: 500m
+        - Desired CPU limit from Turbo analysis: 2600m
+```
+
+Turbo analysis will generate **executable** resizing up action from 500m to 2600m on this container
+
+## Container Executable Resizing Up to the Best Value Permitted by NS Quota
+
+Then reduce namespace resource quota to 2000m:
```

Comment:
Would this one be better modeled as two separate scenarios? It's a bit weird to have a single scenario that you're expected to modify to get two different results. Is there a reason why we wouldn't want one scenario for the first case and one scenario for the second case? If I only want to test the second case, I have to deploy the scenario which models the first case and then edit it to produce the second case, which I feel like is sort of against the idea of having this repo as a "source of truth" for the scenarios we want to model.

Reply:
Thanks! Updated to 2 separated scenarios